### PR TITLE
Bugs/create org for aatf bugs

### PR DESCRIPTION
**Pull request by willdot**
Missing bugs for Mansi

### CODE REVIEW

**Comment by willdot** [comment]
> Can we just add unit test to ensure the breadcrumb it set

There are tests at the bottom of AddAatfControllerTests (When I looked at the commit, it doesn't load them as "Large diffs are not rendered by default")

**Comment by JohnPadgettCivica** [comment]
> > Can we just add unit test to ensure the breadcrumb it set
> 
> There are tests at the bottom of AddAatfControllerTests (When I looked at the commit, it doesn't load them as "Large diffs are not rendered by default")

Just me being lazy then, doh

**Comment by JohnPadgettCivica** [comment]
Think just needs a merge from develop...

**Comment by willdot** [comment]
> Think just needs a merge from develop...

Ha I glanced over it as well at first. Merge with dev done. Wanna just check the change and then merge please :)

File: src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs [hunk 1]
```diff
@@ -87,7 +83,11 @@ public async Task<ActionResult> ManageAatfs(ManageAatfsViewModel viewModel)
             {
                 using (var client = apiClient())
                 {
-                    viewModel.AatfDataList = await GetAatfs(viewModel.Filter);
+                    viewModel = new ManageAatfsViewModel
+                    {
+                        AatfDataList = await GetAatfs(viewModel.Filter),
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs]: Dunno if its me being blind again but cant see a test that sets the aatfDataList property of the viewmodel

Comment (willdot) [reply src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs]: Um test ManageAatfsPost_ModelError_GetAatfsMustBeRun does a check that the call to get aatfs is called.

Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs]: I think we are ok, seems covered elsewhere... one of those days it seems

**Pull request merged**